src/components/inputs/date/date.component.tsx

```diff
@@ -54,6 +54,7 @@ const DateField: React.FC<FormFieldInputProps> = ({ field, value: dateValue, err
 
   useEffect(() => {
     if (dateValue) {
+      setFieldValue(dateValue);
```

Comment:
On initialization, the date field is set to the current date but this value wasn't being set so validation and reusing the value in other parts of the form wasn't possible because it was null